client/app/components/MatchScorePreview.tsx

```diff
@@ -0,0 +1,92 @@
+import { Pressable, Image, View, Text, ViewStyle, TextStyle, ImageStyle } from "react-native";
+import { MatchScorePreviewType } from "./MatchScorePreviewTempData/MatchScorePreviewData";
+
+// Mock interface this way we save time when linking up client and api.
+interface MatchScorePreviewProps {
+  matchDetails: MatchScorePreviewType;
+}
+
+export const MatchScorePreview: React.FC<MatchScorePreviewProps> = ({ matchDetails }) => {
+  {/* Each component gets its data from the matchDetails prop */}
+  return (
+    <Pressable style={$mainContainer}>
+      <Image style={$teamLogo} source={matchDetails.team1ImageSource} resizeMode="contain"/>
+      <View style={[$detailsContainer, {paddingRight: 5, marginLeft: -10}]}>
+        {/* Need to change so text overflows */}
+        <Text style={[$teamNameTextStyle, {textAlign: "right"}]} numberOfLines={1}>{matchDetails.team1Name}</Text>
```

Comment:
Oh yeah, that would be a good idea. I think we should probably do that once we link up the API it might be confusing right now with not having users linked up. And I don't think fantasy apps use this but because we are more doing a daily fantasy style this makes sense to do.